packages/elements/src/internals/machines/sign-up/router.types.ts

```diff
@@ -131,4 +131,12 @@ export type SignInRouterMachineActorRef = ActorRefFrom<TSignUpRouterParentMachin
 
 // ---------------------------------- Snapshot ---------------------------------- //
 
-export type SignUpRouterSnapshot = SnapshotFrom<TSignUpRouterParentMachine>;
+export type SignUpRouterSnapshot = MachineSnapshot<
+  SignUpRouterContext,
+  SignUpRouterEvents,
+  SignUpRouterChildren,
+  SignUpRouterStateValue,
+  SignUpRouterTags,
+  SignUpRouterOuptut,
+  any // TMeta - Introduced in XState 5.12.x
+>;
```

Comment:
Updates to match signin router types https://github.com/clerk/javascript/blob/168607a569cb6d4be3f93c5409b6aa6606e9580b/packages/elements/src/internals/machines/sign-in/router.types.ts#L134-L142